upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -138,6 +242,16 @@ spec:
         prometheus.io/port: "8085"
         prometheus.io/scrape: "true"
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - cluster-autoscaler
+            topologyKey: kubernetes.io/hostname
```

Comment:
As mentioned elsewhere, would like a soft anti-affinity to zone, to handle the 5 nodes in 3 zones case.

Reply:
I think we could change `topologyKey` to `topology.kuberenetes.io/zone`. It wouldn't cover if someone puts all 3 CP nodes in the same zone. But I wonder `ControlPlaneControllerReplicas` should then return 1.

Reply:
It still makes sense to have 2 replicas in a single-zone multi-control-plane-node cluster.